api/proto/file/v1/file.proto

```diff
@@ -88,15 +92,43 @@ message LoadTestDataResponse {
   repeated string error_path = 1;
 }
 
+message ReadRequest {
+  protosupport.v1.IdRaw id = 1;
+  // The length of the bytes you want to read
+  int32 buf_size = 2;
+}
+
+message ReadResponse {
+  // This id might contain an error value.
+  protosupport.v1.IdRaw id = 1;
+  bytes buf = 2;
+  bool truncated = 3;
```

Comment:
What does it mean if the read was truncated?

Reply:
We return a few bytes to the host to ensure low latency. If the host does not get as many bytes as it requested, the truncation is true. If they also ask for very few bytes, they may get the entire bytes they want. In this case, the truncation value is false. Does that make sense?